app/controllers/sitemap_controller.rb

```diff
@@ -0,0 +1,47 @@
+class SitemapController < ApplicationController
```

Comment:
Are you planning to use the Metal controller? If you do, you can use `request` object to access the current community like this: https://github.com/sharetribe/sharetribe/blob/master/app/controllers/landing_page_controller.rb#L188